apps/sheriff-webservices/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "tsx watch ./src/index.ts",
     "build": "tsc",
```

Comment:
but then we have to do `noEmit=false` here

Reply:
Nope! `sheriff-webservices` extends `@tsconfig/node-lts/tsconfig.json` (which might not be intentional) which doesn't set noEmit.

Reply:
oh, you are right